packages/ods/src/style/_range.scss

```diff
@@ -0,0 +1,156 @@
+    border-width: 0;
+    border-radius: var(--ods-border-radius-sm);
+    background: linear-gradient($range-background-color-active, $range-background-color-active) 0 / var(--ods-range-value-percentage) 100% no-repeat, $range-background-color;
+
+    &:disabled {
+      background: linear-gradient(var(--ods-color-neutral-600), var(--ods-color-neutral-600)) 0 / var(--ods-range-value-percentage) 100% no-repeat, $range-background-color;
+    }
+  }
+
+  &:disabled {
+    cursor: not-allowed;
+
+    &::-webkit-slider-thumb {
+      cursor: not-allowed;
+    }
+
+    &::-moz-range-thumb {
+      cursor: not-allowed;
+    }
+
+    &::-ms-thumb {
+      cursor: not-allowed;
+    }
+  }
+
+  &:not(:disabled)#{$range}--error {
+    &::-webkit-slider-thumb {
+      border-color: var(--ods-form-element-border-color-critical);
+    }
+
+    &::-moz-range-thumb {
+      border-color: var(--ods-form-element-border-color-critical);
+    }
+
+    &::-ms-thumb {
+      border-color: var(--ods-form-element-border-color-critical);
+    }
+  }
+}
+
+@mixin thumb() {
+  appearance: none;
+  box-sizing: border-box;
+  border: 1px solid var(--ods-color-neutral-300);
+  border-radius: 50%;
+  box-shadow: none;
+  background-color: var(--ods-color-neutral-000);
+  cursor: pointer;
+  width: $ods-input-range-size-thumb;
+  height: $ods-input-range-size-thumb;
+  pointer-events: auto;
+}
+
+@mixin thumb-hover-active() {
+  &:active {
+    border-color: var(--ods-color-primary-200);
+    background-color: var(--ods-color-primary-200);
+  }
+
+  &:hover {
+    border-color: var(--ods-color-primary-200);
+    background-color: var(--ods-color-primary-200);
+  }
+}
+
+
+@mixin ods-range-dual() {
+  .ods-range__range-dual {
+    z-index: 1;
+    background: linear-gradient(to right,
+      $range-background-color 0,
+      $range-background-color min(var(--ods-range-value-percentage), var(--ods-range-value-percentage-dual)),
+      $range-background-color-active min(var(--ods-range-value-percentage), var(--ods-range-value-percentage-dual)),
+      $range-background-color-active max(var(--ods-range-value-percentage), var(--ods-range-value-percentage-dual)),
+      $range-background-color max(var(--ods-range-value-percentage), var(--ods-range-value-percentage-dual)),
+      $range-background-color 100%);
+
+    &:disabled {
+      background: linear-gradient(to right,
+        $range-background-color 0,
+        $range-background-color min(var(--ods-range-value-percentage), var(--ods-range-value-percentage-dual)),
+        var(--ods-color-neutral-600) min(var(--ods-range-value-percentage), var(--ods-range-value-percentage-dual)),
+        var(--ods-color-neutral-600) max(var(--ods-range-value-percentage), var(--ods-range-value-percentage-dual)),
+        $range-background-color max(var(--ods-range-value-percentage), var(--ods-range-value-percentage-dual)),
+        $range-background-color 100%);
+    }
+  }
+
+  .ods-range__range {
+    position: absolute;
+    z-index: 2;
```

Comment:
if this value is actually meant to be higher than the `z-index: 1` of `.ods-range__range-dual`, use a variable on the latest and set this to `$the-var + 1`